src/common/src/types/to_sql.rs

```diff
@@ -58,3 +59,43 @@ impl ToSql for ScalarImpl {
         true
     }
 }
+
+impl ToSql for ScalarRefImpl<'_> {
```

Comment:
We support `ToSql` on `ScalarRefImpl`, so that we can directly iterate on `RowRef` of the input chunks.

Reply:
This is **unused** as of now. But in subsequent PR when we start optimizing the PG sink, we can use this when scalar adapter is not rrequired.